spring-security-web-app/README.md

```diff
@@ -0,0 +1,225 @@
+---
+page_type: sample
+languages:
+- java
+- html
+author: ramya25
+products:
+- spring security
+- azure-active-directory
+description: "This sample demonstrates a Java web application showcasing how to use Spring security to validate the access token obtained using OAuth2.0."
+urlFragment: ms-identity-java-webapp
+---
+
+# A Java web application using Spring security which signs in users with the Microsoft identity platform
+
+## About this sample
+
+### Overview
+
+This sample demonstrates a Java web application showcasing how to use spring security to validate the access token obtained using OAuth2.0 and authenticates the user.
```

Comment:
I do not see any token validation happening in this sample. I think this sample demonstrates signing in or authenticating a user with Azure AD using Spring Security and OAuth 2.0.

Reply:
No token validation, just using Spring Security for logging in an user via Oauth2

Reply:
updated